packages/wallet/api/src/lib-wallet.js

```diff
@@ -556,11 +557,42 @@ export function makeWallet({
       });
   }
 
+  /**
+   * @param { () => ( Promise | undefined ) } get - The function whose return value
+   * to memoize.
+   */
+  const makeMemoizedGetter = get => {
+    let pk;
+    const doGet = () => pk.resolve(get());
+
+    return () => {
+      if (pk === undefined) {
+        pk = makePromiseKit();
+        doGet();
+      }
+      return pk.promise;
+    };
+  };
+
+  const getAttIssuer = makeMemoizedGetter(() =>
+    E(agoricNames)?.lookup('issuer', 'Attestation'),
+  );
+
+  const getAttBrand = makeMemoizedGetter(() => E(getAttIssuer()).getBrand());
+
+  /** @type import('@endo/promise-kit').PromiseKit<AttestationTool> */
+  const attMakerPK = makePromiseKit();
+
+  const getAccountState = async () => E(attMakerPK.promise).getAccountState();
```

Comment:
This isn't an async function. It's a sync function that returns a promise. I think it doesn't even need a function definition:
```
const { getAccountState } = E(…
```

Reply:
Nice, fixed